packages/client-search/src/types/MultipleQueriesQuery.ts

```diff
@@ -35,5 +35,9 @@ export type MultipleQueriesQuery = SharedMultipleQueriesQuery &
          * The facet name.
          */
         readonly facet: string;
+        /**
+         * The search options.
+         */
+        readonly params?: SharedMultipleQueriesQuery['params'] & { readonly facetQuery?: string };
```

Comment:
```suggestion
        readonly params?: SharedMultipleQueriesQuery['params'] & {
        /**
         * The search query used to search the facet attribute. Follows the same rules for an index query: a single character, a partial word, a word, or a phrase.
        */
        readonly facetQuery?: string
        };
```

but formatted

Reply:
added in https://github.com/algolia/algoliasearch-client-javascript/pull/1267/commits/b6e8506c0d98c4275a3489415a3574abf13bf12e